providers/resteasy-validator-provider-11/src/main/java/org/jboss/resteasy/plugins/validation/GeneralValidatorImpl.java

```
@@ -479,8 +497,22 @@ protected boolean parametersResolveToSameTypes(Method subTypeMethod, Method supe
       ResolvedType resolvedSubType = typeResolver.resolve(subTypeMethod.getDeclaringClass());
       MemberResolver memberResolver = new MemberResolver(typeResolver);
       memberResolver.setMethodFilter(new SimpleMethodFilter(subTypeMethod, superTypeMethod));
-      ResolvedTypeWithMembers typeWithMembers = memberResolver.resolve(resolvedSubType, null, null);
-      ResolvedMethod[] resolvedMethods = typeWithMembers.getMemberMethods();
+      final ResolvedTypeWithMembers typeWithMembers = memberResolver.resolve(resolvedSubType, null, null);
+      ResolvedMethod[] resolvedMethods = new ResolvedMethod[0];
+      try {
+         if (System.getSecurityManager() == null) {
+            resolvedMethods = typeWithMembers.getMemberMethods();
+         } else {
+            resolvedMethods = AccessController.doPrivileged(new PrivilegedExceptionAction<ResolvedMethod[]>() {
+               @Override
+               public ResolvedMethod[] run() throws Exception {
+                  return typeWithMembers.getMemberMethods();
+               }
+            });
+         }
+      } catch (PrivilegedActionException pae) {
+         LogMessages.LOGGER.error(pae.getMessage());

```

I think it should throw exception instead of logging it.